### ISSUE TITLE
Custom global actions support

### ISSUE THREAD

**Issue by sobolevn** [smpallen99/ex_admin#83]
I would like to suggest to add a layout to support custom global actions. Here's the usecases:

1. Logout (my usecase at the moment)
2. Jump to the main site
3. Report a problem
4. View docs
5. Change password
6. View profile

Or any others defined by the developer. It is possible to add them on the separate `div` under the `<div class="header" id="header">...</div>`. Or to the right-end of it.

Here's what `Django` has: 

<img width="1280" alt="2016-05-03 15 18 32" src="https://cloud.githubusercontent.com/assets/4660275/14982886/5fae2bfe-1142-11e6-9c4f-d14d54d4c1d1.png">

**Comment by sobolevn** [comment]
Well, I have found this new demo: http://demo2.exadmin.info/admin
It probably partially solves my question with `Logout` button. But it's still not clear about other possible actions.

**Comment by smpallen99** [comment]
Good idea. I'm wondering if this functionality should be added to config? The other option would be to add a separate with an appropriate DSL. At this time, I think config would be the way to go. Something like:
```
config :ex_admin, 
  header_actions: [
    "<a href='/log-out'>Logout</a>",
   "<a href='http://hex.pm/docs/ex_admin'>Docs</a>"
  ]
```

Might also be nice to support a template:

```
config :ex_admin, 
  header_actions: [
    template: {MyApp.AdminView, "header-actions.html"}
  ]
```
If we are doing this, then we should also support pull down menu item too...

Need to think about it a bit more.

@sobolevn Is this what you were thinking of?

**Comment by sobolevn** [comment]
Yes, you got my idea right. It don't think, that putting `html` in config is nice. But I like the `DSL`-way.

**Comment by smpallen99** [comment]
After more thought, I don't think a DSL is the way to go. At this point ExAdmin is not stateful except for the html dsl builder. Adding a new file with DSL would require state management that I don't think would any value over config. 

I can see how some might not want to put html in the config file, but its not really an issue for me since I've done this several projects where I want to offer some minor branding configuration. 

So, for those that don't like html in the config, we can support the template approach I presented above. If we also support the text option, then we can leave it up to the programmer on which approach they want to use. 

The other option is to support issue #77 by allowing the programmer to copy and edit the layout file. This is not a trivial enhancement though...

Steve

**Comment by sobolevn** [comment]
Well, I got your point. I can help you with adding `config` options. 
What branch is the latest one?

**Comment by smpallen99** [comment]
@sobolevn I've been developing on `themes` branch. After finished and pushed, I merge it into master. I'm tying to clean up the static files for the new themes. After I finish that, I'm going to move it to 0.7.4 branch and publish on hex. 

Short answer `themes` branch :)
